pkg/tbtc/maintainer/local_bitcoin_chain_test.go

```diff
@@ -0,0 +1,88 @@
+package maintainer
```

Comment:
Let's rename it to `bitcoin_chain_test.go`. The `local` prefix is redundant as this is a test file that suggests we are not dealing with a real implementation.

Reply:
Done in 70604b7878805b61aa7b60325d603d656c3799c8.